arelle/plugin/validate/ESEF/Const.py

```diff
@@ -177,249 +177,261 @@
 )
 
 esefMandatoryElementNames2022 = (
-    "AddressOfRegisteredOfficeOfEntity",
```

Comment:
Why the change from 4-space indent to 3-space indent?  That seems inconsistent with the rest of the repo.